client/libs/ga-scroll-depth.js

```diff
@@ -1,29 +1,33 @@
 // Simplified/rewritten from https://github.com/leighmcculloch/gascrolldepth.js
-
 import { onWindowScrollDebounce$ } from '../js/utils/dom-events';
+import getFlags from '../js/components/launchdarkly';
+
 const startTime = new Date().getTime();
 const getElHeight = el => el.offsetHeight + el.offsetTop;
 
 const getWindowHeight = () => {
   return window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
 };
 
-const sendEvent = (distance, timing) => {
-  window.ga('send', {
-    hitType: 'event',
-    eventCategory: 'Scroll Depth',
-    eventAction: 'Percentage',
-    eventLabel: distance,
-    eventValue: 1,
-    eventNonInteraction: true
-  });
-
-  window.ga('send', {
-    hitType: 'timing',
-    timingCategory: 'Scroll Timing',
-    timingVar: `Milliseconds to ${distance}`,
-    timingValue: timing
-  });
+const sendEvent = (distance, timing, flags) => {
+  if (flags['ga-scroll-depth']) {
```

Comment:
WHy not move the `flags.then` right up to where they're needed? It feels a little like it's in purgatory at the moment.

I'd say either the module relies on it (this module does actually do anything if the flags are off) or this method relies on it as side effect... just seems easier to fathom.